app/src/main/java/com/google/jetpackcamera/MainActivity.kt

```diff
@@ -65,6 +72,19 @@ class MainActivity : ComponentActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        val externalContentValues: ContentValues? =
+            if (intent.extras == null) null
```

Comment:
nit: add the braces for if 

https://developer.android.com/kotlin/style-guide#braces